src/org/zaproxy/zap/extension/httpsinfo/ZapAddOn.xml

```diff
@@ -2,16 +2,19 @@
 	<name>HttpsInfo</name>
 	<version>7</version>
 	<status>alpha</status>
-	<description>Adds Https status information</description>
+	<description>Adds a context menu to the site tree and history table use of 
+	which displays HTTPS status information, scans for and 
+	alerts on some HTTPS vulnerabilities.</description>
```

Comment:
Yeah, that needs to be improved. You could try adding the break lines manually, like we do for the "changes" entry.